packages/trader/src/Modules/Trading/Components/Form/TradeParams/Multiplier/take-profit.jsx

```diff
@@ -32,6 +32,9 @@ const TakeProfit = observer(props => {
             onChange(e);
         }
     };
+    const tooltip_text = is_multiplier
```

Comment:
As Take profit component is also used by Accumulators and Turbos, I added check for Multipliers